deploy/helm/kubecf/values.yaml

```
@@ -238,6 +238,12 @@ features:
     registry:
       service:
         nodePort: 32123
+    rbac:
+      cluster_role:
+        eirini_webhook:
+          rules:
+            resources: ~
+            verbs: ~
```

As I suggested on https://github.com/cloudfoundry-incubator/kubecf/issues/595, you can use `rules` as an array. The goal is to match what the user provides here and what the Kubernetes API expects.

```suggestion
          rules: []
```

---

Thanks, changed.